crates/workspace/src/workspace_settings.rs

```diff
@@ -75,6 +76,11 @@ pub struct WorkspaceSettingsContent {
     ///
     /// Default: off
     pub autosave: Option<AutosaveSetting>,
+    // Maximum open tabs in a pane. Will not close an unsaved
+    // tab. Set to `0` for unlimited tabs.
+    //
+    // Default: `5`
```

Comment:
I think you put 0 as default, didn't you? And I'm not sure how tests could be passing seeing that you didn't set the tab cap..